client/src/index.html

```diff
@@ -108,7 +108,7 @@ <h6 class="sidebar-heading d-flex justify-content-between align-items-center px-
 
         <main role="main" class="col-md-9 ml-sm-auto col-lg-10 px-4"><div class="chartjs-size-monitor" style="position: absolute; left: 0px; top: 0px; right: 0px; bottom: 0px; overflow: hidden; pointer-events: none; visibility: hidden; z-index: -1;"><div class="chartjs-size-monitor-expand" style="position:absolute;left:0;top:0;right:0;bottom:0;overflow:hidden;pointer-events:none;visibility:hidden;z-index:-1;"><div style="position:absolute;width:1000000px;height:1000000px;left:0;top:0"></div></div><div class="chartjs-size-monitor-shrink" style="position:absolute;left:0;top:0;right:0;bottom:0;overflow:hidden;pointer-events:none;visibility:hidden;z-index:-1;"><div style="position:absolute;width:200%;height:200%;left:0; top:0"></div></div></div>
           <div class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pt-3 pb-2 mb-3 border-bottom">
-            <h1 class="h2">Trello KPI Dashboard</h1>
+            <h1 class="h2"><p id="axios_error">Trello KPI Dashboard</p></h1>
```

Comment:
Normally you wouldn't swap a header. You might want to add an empty div with this id. <div id="axios_error"></div>

Reply:
I think that my new solution for this is better.  It's hard to put things into the canvas on the template that is designed for the chart, so I am just using the chart title.  That way it is in the correct element.
![fail](https://user-images.githubusercontent.com/29137994/49668133-d984ef00-fa22-11e8-8edc-6c376a6ff85e.png)